src/Parser/DeathKnight/Blood/Modules/Features/BoneShield.js

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import { formatNumber, formatPercentage } from 'common/format';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import StatTracker from 'Parser/Core/Modules/StatTracker';
+
+class BoneShield extends Analyzer {
+
+  static dependencies = {
+    combatants: Combatants,
+    statTracker: StatTracker,
+  };
+
+  SS_DR = 0.08;
+  BONE_SHIELD_DR = 0.16;
+
+  EXCLUDES_ABSORBS = [
+    SPELLS.BLOOD_SHIELD.id,
+    SPELLS.BLOOD_MIRROR_TALENT.id,
+  ];
+
+  totalDamageTaken = 0;
+  excludedAbsorbs = 0;
+
+  hasSS = false;
+  hasSD = false;
+
+  totalCrit = 0;
+  totalCritChecks = 0;
+
+  boneShieldDR = 0.16;
+  ssDR = 0;
+
+
+  on_initialized() {
+    //ToDo: create Spell
+    this.hasSS = this.combatants.selected.traitsBySpellId[192558];
+    this.hasSD = this.combatants.selected.hasTalent(SPELLS.SPECTRAL_DEFLECTION_TALENT.id);
+
+    console.info(this.hasSD);
+  }
+
+  getBoneShieldAbsorbTooltip() {
```

Comment:
Minor thing, but you could change this and `getSkeletalShatteringTooltip()` to be actual getters `get skeletalShatteringTooltip()` since they don't take parameters anyway.